### ISSUE TITLE
Hamburger not contrasted enough

### ISSUE THREAD

**Issue by VickyRampin** [nyudlts/ultraviolet#158]
Currently in our test instance of UltraViolet, the hamburger menu is black on purple:

![Screenshot_20221215_122718](https://user-images.githubusercontent.com/10644777/207948401-d0034029-831f-4a0f-aed5-ed7a9798f6de.png)

I legit could not find it for a while - in my opinion it would be better as white to be more contasted against the page and easier to find:

![Screenshot_20221215_123010](https://user-images.githubusercontent.com/10644777/207948502-106d6dd0-26ca-499b-b964-2ca61fd4108b.png)

I got that to happen in the inspector by changing the `color` to white as so:

```
#rdm-burger-menu-icon .navicon::after, #rdm-burger-menu-icon .navicon::before, #rdm-close-burger-menu-icon .navicon::after, #rdm-close-burger-menu-icon .navicon::before {color#fff;}
#rdm-burger-menu-icon .navicon, #rdm-close-burger-menu-icon .navicon {color:#fff}
```

I can actually find the right CSS files to change this but just wanted to bring it up for discussion before proposing a change via PR.

**Comment by ekate** [comment]
Sorry, I think it is regression (Laura already fixed it earlier).Will track
and fix

On Thu, Dec 15, 2022 at 2:22 PM Vicky Rampin (née Steeves) <
***@***.***> wrote:

> Currently in our test instance of UltraViolet, the hamburger menu is black
> on purple:
>
> [image: Screenshot_20221215_122718]
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__user-2Dimages.githubusercontent.com_10644777_207948401-2Dd0034029-2D831f-2D4a0f-2Daed5-2Ded7a9798f6de.png&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=DI-uvznCdEKqKEynFrsCGw&m=IPwPSDrS7gzu__fo2kswFihhue4RAjwP9ah-FWX6d6-Lrznkzp8cp4AVZOmXqb1H&s=md78xdGd4NcnkuZSc7MYXrcTjFh8eOtKQ73eXzf3PQc&e=>
>
> I legit could not find it for a while - in my opinion it would be better
> as white to be more contasted against the page and easier to find:
>
> [image: Screenshot_20221215_123010]
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__user-2Dimages.githubusercontent.com_10644777_207948502-2D106d6dd0-2D26ca-2D499b-2Db964-2D2ca61fd4108b.png&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=DI-uvznCdEKqKEynFrsCGw&m=IPwPSDrS7gzu__fo2kswFihhue4RAjwP9ah-FWX6d6-Lrznkzp8cp4AVZOmXqb1H&s=II2w6mhSIGD8DHoymCRQGmvMqg9I0xt0KuBT-6pCDtQ&e=>
>
> I got that to happen in the inspector by changing the color to white as
> so:
>
> #rdm-burger-menu-icon .navicon::after, #rdm-burger-menu-icon .navicon::before, #rdm-close-burger-menu-icon .navicon::after, #rdm-close-burger-menu-icon .navicon::before {color#fff;}
> #rdm-burger-menu-icon .navicon, #rdm-close-burger-menu-icon .navicon {color:#fff}
>
> I can actually find the right CSS files to change this but just wanted to
> bring it up for discussion before proposing a change via PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_nyudlts_ultraviolet_issues_158&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=DI-uvznCdEKqKEynFrsCGw&m=IPwPSDrS7gzu__fo2kswFihhue4RAjwP9ah-FWX6d6-Lrznkzp8cp4AVZOmXqb1H&s=hFjNmcO__SO-Aldgm7LnXDKdrWEDyYHUpRuU2m54btc&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AATL7ZUJWMBF4DPB4UKBTRTWNNVYRANCNFSM6AAAAAATAC2DPE&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=DI-uvznCdEKqKEynFrsCGw&m=IPwPSDrS7gzu__fo2kswFihhue4RAjwP9ah-FWX6d6-Lrznkzp8cp4AVZOmXqb1H&s=5wOWuDIjhqnW63eOyESZSuNjk4JAL9mz38fqIeC7Ahs&e=>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Ekaterina Pechekhonova
Senior DevOps Engineer
New York University
Libraries
email: ***@***.***
phone: 212-992-9993

**Comment by debverhoff** [comment]
This was resolved a while ago. Closing ticket.
